rs/execution_environment/src/query_handler.rs

```diff
@@ -101,6 +103,7 @@ pub struct InternalHttpQueryHandler {
     log: ReplicaLogger,
     hypervisor: Arc<Hypervisor>,
     own_subnet_type: SubnetType,
+    own_subnet_id: SubnetId,
```

Comment:
Seems we have the subnet ID inside the hypervisor?

Reply:
Done.